release_scripts/signassert.py

```diff
@@ -0,0 +1,63 @@
+################################################################################
+#                                                                              #
+# Copyright (C) 2011-2015, Armory Technologies, Inc.                           #
+# Distributed under the GNU Affero General Public License (AGPL v3)            #
+# See LICENSE or http://www.gnu.org/licenses/agpl.html                         #
+#                                                                              #
+################################################################################
+
+import getpass
+import os
+import sys
+sys.path.append(os.path.join(os.path.dirname(os.path.abspath(__file__)), '..'))
+
+from armoryengine.ALL import *
```

Comment:
Okay. I think I remember having the same problem myself awhile back. Unfortunately, the "import *" line causes a lot of trouble in that regard. The code probably should've been more carefully constructed in that regard from the getgo. But, as they say, if wishes were fishes....

I'm wondering if it would be possible to move all the argparse stuff into its own separate file and have only ArmoryUtils import it. I don't think that would work but I'm curious nonetheless. Maybe take a minute and give it a go? If it breaks major things or doesn't seem to work, don't worry about it.